crypto/BRCryptoWalletManager.c

```diff
@@ -269,6 +275,12 @@ cryptoWalletManagerCreate (BRCryptoCWMListener listener,
         }
     }
 
+cleanupOnError:
```

Comment:
Yikes! Goto statements? These can't be avoided? These would be the first in our source.

Reply:
I know; maybe the first in my life:
```
static void
doCleanupOnWalletManagerError (BRCryptoWalletManager *cwmPtr) {
  cryptoWalletMangerGive (*cwmPtr);
  *cwmPtr = NULL;
}
```
and then
```
if (NULL = ....) { doCleanupOnWalletManagerError(&cwm); break; }
```
? 

Or goto?

Reply:
Or just do (in the BTC case statement):
```
BRWalletManager bwm = BRWalletManagerNew();
if (NULL != bwm) {
  // normal business
} else {
  // error handling
}
```
Adds one level of indentation, but removes need for goto?

Reply:
Updated.